packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -79,6 +79,15 @@
 TransactionType = TypeVar("TransactionType")
 
 
+def get_name(prop: Any) -> str:
```

Comment:
Would `mypy` warn about this? 

```suggestion
def get_name(prop: property) -> str:
```